### PR TITLE
Beluga-Class Transport (we rich as hell!)

### PR DESCRIPTION
**Pull request by spockye**
## About The Pull Request
so, what is the Beluga class?
well! its a high-class transport ship, used by the rich and important. its sleek design, rich interior and high speed makes it perfect for transportation of high value passengers. 

While the ship has no science materials, barely any medical supplies, almost no security gear or engineering supplies, it DOES have a fancy kitchen, a rich interior design and a total of...      6 cryo pods!!! (rich people don't SHARE cryopods???)

it exists to fill the niche of "rich person roleplay", something that was left empty after the removal of the high class

----

the ship contains:
two bedrooms
a bunk for the crew
a large meeting room
a kitchen/bar
a inteq security checkpoint
and other essentials!

----

The crew:
1x Captain
1x Bridge officer
2x Inteq private security officers
1x bartender
1x mechanic
1x janitor
4x high class passenger

image

image

image

## Why It's Good For The Game
its a unique ship that fills a currently un-used niche, and it builds onto the lore by having hired inteq guards

## Changelog

:cl:
add: Adds the Beluga class transport
/:cl:

### CODE REVIEW

**Review by Erikafox — CHANGES_REQUESTED** [review]
This needs an exterior docking port. Ping in ruinyard or look at the Borea

**Comment by AverageUser67** [comment]
one of the bedrooms has a medical laptop the passengers cant access
the rich people have to SHARE bedrooms too

**Comment by AverageUser67** [comment]
also the pure fernet bottle is... Not the best choice of beverage, maybe champagne

**Comment by AverageUser67** [comment]
the pixelshifted rack makes items you put on it go in the default tile center
there's no hatchet to chop the wood logs
this was based on a live play test so it might be different for others idk

**Comment by AverageUser67** [comment]
you CAN just make a hatchet with the autolathe and roundstart materials but that's much better

File: code/modules/jobs/job_types/security_officer.dm [hunk 1]
```diff
@@ -231,6 +231,22 @@ GLOBAL_LIST_INIT(available_depts, list(SEC_DEPT_ENGINEERING, SEC_DEPT_MEDICAL, S
 	courierbag = /obj/item/storage/backpack/messenger/inteq
 	backpack_contents = list(/obj/item/melee/baton/loaded=1, /obj/item/ammo_box/magazine/co9mm=1, /obj/item/gun_voucher=1,/obj/item/pda/security)
 
+/datum/outfit/job/security/inteq/beluga
+	name = "IRMG Enforcer (Inteq)"
```
Comment (Erikafox) [inline code/modules/jobs/job_types/security_officer.dm]: You may want to include a (Beluga) here, as I believe the enforcer is already defined with this name elsewhere.

File: code/modules/jobs/job_types/lawyer.dm [hunk 2]
```diff
@@ -37,3 +37,13 @@
...
 	r_pocket = /obj/item/pen/fountain
+
+/datum/outfit/job/lawyer/passenger
+	uniform = /obj/item/clothing/under/suit/black
+	suit = null
+	ears = /obj/item/radio/headset/headset_cent
+	neck = null
+	glasses = /obj/item/clothing/glasses/sunglasses/big
+	l_hand = null
+	r_pocket = /obj/item/spacecash/bundle/mediumrand
```
Comment (Erikafox) [inline code/modules/jobs/job_types/lawyer.dm:48]: Maybe give them a little more.

**Review by Erikafox — CHANGES_REQUESTED** [review]
image
Please check to see if the sepia tiles still have slowdown, and consider giving the ship some actual planks to fuel the fireplace
StrongDMM_E6WgeBIxi5
move this down 4 pixels please

Please don't use factional secret documents here, If you'd like them to have something unique, I'd subtype something like... "Trade Secrets". Give it that macguffin value.
StrongDMM_WygPbdVyY7
please give these windows their own button inside the high passage room. They're paying customers, they should be able to look outside
StrongDMM_fXOwBUSsHJ
put a req access on this button so only passengers + the captain can open up the private room from outside.
StrongDMM_nZX5G27oRS
Experiment with directions here
StrongDMM_CXXvWgcMaX
Random syringes please, once someone figures out that there's roundstart meth. Well.

No stacking wallmounts like that
image
Consider putting a table wideband up here.
StrongDMM_6k2d0r757p
The curved window must go...

The ship needs newscasters. Like. Several
StrongDMM_3SpcrgIdWZ
random syringes here too
StrongDMM_MZ0JVW44JF
consider windows + shutters on the external part of this airlock. It might look nice.
StrongDMM_oQ4vZ3D4D9
width 30 dwidth 15
StrongDMM_Omu3ty3w7F
I'd remove the food processor and give them a cutting board. Not only is it a ghetto alternative, but it adds some roleplay potential
StrongDMM_nlO7lPpVeI
No catwalk floors.
StrongDMM_QA6GwmMexE
The dock and the subshuttle dock should not overlap. I'd put the dock down in the cargo area.
image
I'd let security toggle this themselves
StrongDMM_g2HPsW3K2l
subtype like. an inteq megaphone or something
StrongDMM_xIKXHgIVcO
Real Stun Batons. Telebatons have instant knockdown, stun batons have a delayed one. 
StrongDMM_lmg0epWBxS
If this is supposed to be a dressing area, give it a door on the south

Sorry this took so long

File: code/modules/jobs/job_types/security_officer.dm (hunk 1 above)
Comment (spockye) [reply code/modules/jobs/job_types/security_officer.dm]: donezo

**Review by Latentish — APPROVED** [review]
This is a good ass ship I've seen it in round, it works, the ship is nice, Approval from me.

Make sure to address Blessed's review.

**Review by Erikafox — APPROVED** [review]

**Pull request merged**